tests/dummy/app/app.js

```diff
@@ -6,6 +6,7 @@ import config from './config/environment';
 let App;
 
 Ember.MODEL_FACTORY_INJECTIONS = true;
+Ember.run.backburner.DEBUG = true;
```

Comment:
AFIK this will only apply to the dummy app, so I don't think it's unreasonable to have this turned on. But yeah, definitely need to make sure this doesn't affect consuming applications.